### PR TITLE
Add standard PATH priors

### PR DESCRIPTION
**Pull request by profxj**
Simple method to enable PATH priors

### CODE REVIEW

File: astropath/path.py [hunk 1]
```diff
@@ -94,14 +94,15 @@ def init_cand_prior(self, P_O_method:str, P_U=0.):
...
         assert priors.vet_cand_prior(self.cand_prior, self.candidates), 'Bad candidate prior input'
 
-    def init_theta_prior(self, PDF:str, max:float):
+    def init_theta_prior(self, PDF:str, max:float, scale:float):
         """Ingest the theta (offset) prior
 
         Args:
             PDF (str): Method
             max (float): Maximum offset allowed
+            scale (float): Scaling of phi [only for exp method, so far]
```
Comment (tarraneh) [inline astropath/path.py:103]: Add example. e.g., "scale = 2.0 for exp(-2*theta/phi)"

File: astropath/priors.py [hunk 2]
```diff
@@ -118,4 +120,35 @@ def vet_theta_prior(theta_prior:dict):
...
\ No newline at end of file
+    return chk
+
+def load_std_priors(nhalf:float=10., theta_max:float=6.):
+    """
+    Load the standard set of priors as defined
+    in the PATH paper
+
+    Args:
+        nhalf (float, optional): Number of half-light radii for analysis. Defaults to 10..
```
Comment (tarraneh) [inline astropath/priors.py:131]: "Defaults to 10.0." Just looks a little cleaner!

File: astropath/priors.py [hunk 3]
```diff
@@ -118,4 +120,35 @@ def vet_theta_prior(theta_prior:dict):
...
+        theta_max (float, optional): Maximum offset in half-light radii.
+            Defaults to 6..
```
Comment (tarraneh) [inline astropath/priors.py:133]: "Defaults to 6.0."

**Review by tarraneh — CHANGES_REQUESTED** [review]
Minor comments for clarity.

File: astropath/path.py (hunk 1 above)
Comment (profxj) [reply astropath/path.py:103]: good suggestion.  added!

File: astropath/priors.py (hunk 2 above)
Comment (profxj) [reply astropath/priors.py:131]: edited

File: astropath/priors.py (hunk 3 above)
Comment (profxj) [reply astropath/priors.py:133]: edited

**Review by tarraneh — APPROVED** [review]
Looks good

**Pull request merged**